pkg/storage/options/options.go

```diff
@@ -23,6 +24,10 @@ func (o *StorageOptions) Validate() []error {
 		return nil
 	}
 
+	if o.Name == memorystorage.StorageName {
```

Comment:
In the storage definition, try not to couple with the concrete storage implementation, we can determine whether `o.ConfigPath == ”“` or ConfigPath type is `*string`.

Reply:
I used to think so.But if storage is mysql,and users do not config configpath `--storage-config`,It will cause error later

Reply:
> I used to think so.But if storage is mysql,and users do not config configpath --storage-config,It will cause error later

We can check if the configPath is empty in `internalStorage.NewStorageFactory` and let the storage layer decide if the configPath is needed.

Reply:
ok,got it